scripts/check_tests_are_captured_in_ci.py

```diff
@@ -29,9 +29,7 @@
 
 
 E2E_TEST_SUITES_THAT_ARE_NOT_RUN_IN_CI = ['full']
-ACCEPTANCE_TEST_SUITES_THAT_ARE_NOT_RUN_IN_CI = [
-    'exploration-editor/manage-exploration-misconceptions'
-]
+ACCEPTANCE_TEST_SUITES_THAT_ARE_NOT_RUN_IN_CI: List[str] = []
```

Comment:
Do we still need this variable?

Reply:
Yep, for future use-cases when we want to omit any tests from CI for any reason.